nomad/acl_endpoint.go

```diff
@@ -2735,3 +2741,144 @@ func (a *ACL) OIDCCompleteAuth(
 	reply.ACLToken = tokenUpsertReply.Tokens[0]
 	return nil
 }
+
+// Login RPC performs non-interactive auth using a given AuthMethod. This method
+// can not be used for OIDC login flow.
+func (a *ACL) Login(args *structs.ACLLoginRequest, reply *structs.ACLCompleteAuthResponse) error {
```

Comment:
Do we want to version gate this RPC? In a situation where one server in the region or one region in a federated cluster has not been updated to a version that includes this, I believe the call will always error.

Reply:
Good shout, I gated to 1.5.2 with a TODO comment in the leader. See [28efef6](https://github.com/hashicorp/nomad/pull/15918/commits/28efef61a282d311e4414d37c4d387e6eacd54f8) if you're ok with the implementation.